src/components/webview/WebviewPanel.ts

```diff
@@ -24,9 +24,34 @@ export abstract class IntuitaWebviewPanel {
 	protected _webviewMounted = false;
 
 	constructor(
-		protected readonly _options: Options,
+		private readonly __options: Options,
 		protected readonly _messageBus: MessageBus,
-	) {
+	) {}
+
+	setTitle = (title: string) => {
+		if (this._panel === null) {
+			return;
+		}
+
+		this._panel.title = title;
```

Comment:
`this._panel?.title = title;`

Reply:
why do we need `?` if we have null check on line 32?

Reply:
@DmytroHryshyn it's just more concise (I care a lot about readability and being able to quickly scan a file with one's eyes)

Reply:
I just checked and what I wanted to do is not possible here.